_aggregations/bucket/sampler.md

```diff
@@ -8,7 +8,7 @@ nav_order: 170
 
 # Sampler aggregations
 
-If you're aggregating over millions of documents, you can use a `sampler` aggregation to reduce its scope to a small sample of documents for a faster response. The `sampler` aggregation selects the samples by top-scoring documents.
+If you're aggregating a very large number of documents, you can use a `sampler` aggregation to reduce its scope to a small sample of documents for a faster response. The `sampler` aggregation selects the samples by top-scoring documents.
```

Comment:
Can we replace "its" with the appropriate possessive noun? Or should it just be "the"?